spec/services/brewery_search_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe BrewerySearch do
+  describe ".find_brewery" do
+    it "returns the first brewery with the name sent in the search parameter" do
+      brewery_name = "Anchor"
+
+      stub_request(:any, "https://brianiswu-open-brewery-db-v1.p.rapidapi.com/breweries/search?query=#{brewery_name}").
```

Comment:
Ignoring this for now, because I am getting conflicting messages from Hound and Rubocop about `Layout/DotPosition: Place the . on the next line, together with the method name.`. Choosing to ignore Metrics.LineLength for readability.